### DISCUSSION TITLE
Webhooks aren't firing

### DISCUSSION THREAD

**Discussion by russelltaga** [community/community#24505] [Apps, API and Webhooks]
<p>Similar to this issue reported for actions, we’re not seeing any webhooks fire for different events like commits.</p>
<aside class="quote" data-post="3" data-topic="122894">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/pushing-new-branch-fails-to-start-action-ci-jobs/122894/3">Pushing new branch fails to start Action CI jobs</a> <a class="badge-wrapper  bullet" href="/c/code-to-cloud/github-actions/41"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #2188ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Actions—automation for all of your development workflows, now with world-class CI/CD.">GitHub Actions</span></a>
  </div>
  <blockquote>
    Same here, github action is not being triggered on commit in multiple private repos.
  </blockquote>
</aside>

<p>In our specific case, the webhooks should post to CircleCI.</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>:wave: hello there <a class="mention" href="/russelltaga">@russelltaga</a>, and welcome to the GitHub Support Community!</p>
<p>It looks like the earlier referenced <a href="https://github.community/t/pushing-new-branch-fails-to-start-action-ci-jobs/122894/7">topic</a>’s event was due to a <a href="https://www.githubstatus.com/">GitHub.com status event in our availability</a>. If your webhooks still aren’t firing, would you mind sharing a URL here so we can investigate further?</p>

**Comment by russelltaga** [comment]
<p>Thanks so much for the follow up, issue was indeed yesterday’s fun. Everything works great again we’re all set!</p>
